server/parsec/asgi/rpc.py

```diff
@@ -296,13 +315,14 @@ def _handshake_abort_bad_content(api_version: ApiVersion) -> NoReturn:
 
 @dataclass
 class ParsedAuthHeaders:
-    organization_id: OrganizationID
+    organization_id: OrganizationID | None  # None only if account family
```

Comment:
It seems a bit risky to do that. 

Maybe a separated ParsecAccountHeaders class would be necessary